README.md

```diff
@@ -40,33 +39,37 @@ truncated('selector', maxHeight);
 ```javascript
 truncated('selector:not([not this selector])', maxHeight);
 ```
+**With a special symbol**
+```javascript
+truncated('selector', maxHeight, '🍻');
+```
 
-### How?
+**With a special CSS Class**
+```javascript
+truncated('selector', maxHeight, '🙌', 'your-css-class');
+```
 
-**Truncated.js** trims a text string to a last full word of what can fit within a specified max height.
+**Or with jQuery**
+```javascript
+$('selector').truncated(maxHeight);
+```
 
-### Why?
+## How?
 
-**Truncated.js** is made simpicity when it comes to text truncation. Several plugins that I've looked at oversolve what people want when they want text truncation - _for text to look nice in a specified space_. Here's a basic [example](http://codepen.io/yowainwright/pen/xEwNKJ) & with [jQuery](http://codepen.io/yowainwright/pen/VKvNGE).
+**Truncated.js** trims an element's text string to the last full word that can fit within a specified max height.
 
-This plugin is small - `~1.5kb` unminified & is meant to do 1 thing - _truncate text based on a specified max height_. 
+## Why?
 
-### Options
+**Truncated.js** is made for simpicity; it doesn't try to oversolve truncation or provide tons of options. It truncates text to a max height.
 
-If you'd like to not use the classname 'js-truncated', just use your own. 
+## What about the original text?
 
-```javascript
-reframe('selector', 'classname');
-```
-### jQuery
+**Truncated.js** re-adds the _diff_ of the original seleected elements's text in a hidden `<span>`. It runs a check for that `<span>` to make sure text is truncated as desired every time.
 
-You can use truncated with [jQuery](https://jquery.com/).
+## What about size?
 
-```javascript
-$('selector').reframe(maxheight);
-```
-&, if you'd like to use a custom classname instead of 'js-truncated'
+**Truncated.js** is small - `~1.5kb` unminified and is meant to do 1 thing - _truncate text to fit beatifully to a specified max height_.
```

Comment:
Please do! Thanks! I only merged it because I'm going to be adding it under the DSC namespace!!! 🙌